tests/runtime/out_stackdriver.c

```diff
@@ -16,7 +16,6 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-
```

Comment:
Was this blank line removed intentionally?

In fact, there don't seem to be any non-whitespace changes to this file. Did you plan to make any?

Reply:
I had this opened as I was trying to create tests (which ultimately wouldn't work due to testing environment) and I think post-revert my editor tried to be smart. I will make sure this gets reverted.